spec/lib/jit_preloader/preloader_spec.rb

```diff
@@ -49,6 +48,51 @@
     ->(event, data){ source_map[data[:source]] << data[:association] }
   end
 
+  context "for single table inheritance" do
+    let!(:contact_book) { ContactBook.create(name: "The Yellow Pages") }
```

Comment:
This is an awful lot of data for a spec. Do we actually need all of it?

If so, let's structure the tests so that tests only use the pieces of data that they need. It doesn't look like every spec needs all of the data.

Reply:
The data is used entirely, but each sub-context only uses part of it. I can move them within each context if you would prefer. I had avoided doing that as I didn't want to deviate too much from the existing spec structure.